ui/src/shared/utils/errors.ts

```diff
@@ -12,6 +12,8 @@ if (CLOUD) {
 
 interface AdditionalOptions {
   component?: string
+  context?: object
```

Comment:
any reason this shouldn't be

```ts
context?: {[key: string]: any}
```
?

it's a little more descriptive and makes it easier to limit the values allowed if we find that `any` is too open.